base/_typesizes.scss

```diff
@@ -1,158 +1,43 @@
 //
-// The idea:
-//  • One list of breakpoints.
-//  • One list of type sizes/scales with a corresponding key name.
-//
-// Helpers:
-//  • @include all-type-sizes( size-grouping ): A mixin that can iterate and output all type sizes from one breakpoint
-//  • @incude one-element-size( element ): A mixin that can iterate over the breakpoints, find the
-//	  matching type size, and output all breakpoint sizes for that element.
-// • type-size( element, size-grouping ): A function that can output one type size at one specific breakpoint
+// Required list of sizes for typographics elements
```

Comment:
Should use the same comment style as this page: https://github.com/oomphinc/scss-scaffold/blob/master/base/_mixins.scss